### PR TITLE
fix: reduce log level

### PR DESCRIPTION
**Pull request by joske**
This reduces log level in consensus from info! to debug! in a few cases. 

Rationale: we don't want to have 'per message' logging in info level to avoid log polution.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,5 @@ wasm/Cargo.lock
 **/build
 **.ledger-*
 **/committee/.dev/
+.bft-storage-3/
+validator-*.log
```
Comment (ljedrz) [inline .gitignore:10]: these are unrelated, no?

Comment (joske) [reply .gitignore:10]: not really related to reducing the log level no

File: node/consensus/src/lib.rs [hunk 2]
```diff
@@ -359,7 +359,7 @@ impl<N: Network, C: ConsensusStorage<N>> Consensus<N, C> {
             self.memory_pool.clear_invalid_solutions(self);
         }
 
-        info!("Advanced to block {}", block.height());
+        debug!("Advanced to block {}", block.height());
```
Comment (ljedrz) [inline node/consensus/src/lib.rs]: don't we want to maintain this level? we should be able to see that a block was advanced in every node

File: .gitignore (hunk 1 above)
Comment (joske) [reply .gitignore:10]: can remove if you prefer this separately

Comment (ljedrz) [reply .gitignore:10]: the `validator` bit we don't need anymore IIRC, the other one I'm not sure, so we might want to remove those

**Review by ljedrz — APPROVED** [review]
LGTM with a couple of comments

File: .gitignore (hunk 1 above)
Comment (joske) [reply .gitignore:10]: if you do `run_bft.sh` it creates the `validator-x.log` files.

File: node/consensus/src/lib.rs (hunk 2 above)
Comment (joske) [reply node/consensus/src/lib.rs]: Well, I'm not sure, you can still see it if you want (set verbosity to 1). Not a big deal to me though, this is just one log line per 30 seconds or so.

Comment (ljedrz) [reply node/consensus/src/lib.rs]: yeah, I think I'd keep this as `info` as it's not spammy and it's an interesting piece of information; all the others LGTM
